src/main/java/frc/robot/Constants.java

```diff
@@ -197,6 +197,16 @@ public static class TrapElvConstants {
     public static final double WRIST_LENGTH = Units.inchesToMeters(12.5);
   }
 
+  public static class LimelightConstants {
+    public static final double FIELD_SIZE = (8.308467 * 2);
```

Comment:
This is the field length

Reply:
https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/apriltag/AprilTagFieldLayout.html#getFieldLength() Should we use WPIlib functions to seed these constants?